src/hooks/useSearchParams.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react"
+
+type TUseSearchParams = ( url?: string, opt?: { unique: boolean } ) => Record<string, any>
```

Comment:
I am also getting a lot of problems with the types

<img width="960" alt="image" src="https://github.com/novajslabs/nova.js/assets/63487673/6a693693-4aa7-48e1-a183-9d3f24774f07">

Does not your linter show it to you?